### PR TITLE
misc fixes

### PR DESCRIPTION
**Pull request by deniak**

### CODE REVIEW

File: lib/rules/headers/copyright.js [hunk 1]
```diff
@@ -10,7 +10,7 @@ exports.check = function (sr, done) {
     if ($c.length) {
         var year = (sr.getDocumentDate() || new Date()).getFullYear()
         ,   text = sr.norm($c.text())
-        ,   startRex = "^Copyright © (?:(?:199\\d|20[01]\\d)-)?" + year + " .*?W3C® \\(MIT, ERCIM, Keio, Beihang\\)"
+        ,   startRex = "^Copyright (©|&copy;) (?:(?:199\\d|20[01]\\d)-)?" + year + " .*?W3C(®|&reg) \\(MIT, ERCIM, Keio, Beihang\\)"
```
Comment (tripu) [inline lib/rules/headers/copyright.js]: Missing the semicolon after `reg`.

**Comment by tripu** [comment]
When did we drop the &ldquo;all rights reserved&rdquo; part?

**Comment by tripu** [comment]
@deniak: all good (barring my two comments above)!

File: lib/rules/headers/copyright.js (hunk 1 above)
Comment (deniak) [inline lib/rules/headers/copyright.js]: oups, good catch @tripu

**Comment by deniak** [comment]
> When did we drop the “all rights reserved” part?

That sentence was allowed only until [8 March 2016](https://lists.w3.org/Archives/Public/spec-prod/2015JanMar/0001.html).

**Pull request merged**